config/monitoring/grafana-dashboards/grafana-dashboard-simple-build-pipeline.json

```diff
@@ -0,0 +1,332 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "id": 13,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              }
+            ]
+          }
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 6,
+        "w": 5,
+        "x": 0,
+        "y": 0
+      },
+      "id": 6,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "textMode": "auto"
+      },
+      "pluginVersion": "9.1.6",
+      "targets": [
+        {
+          "editorMode": "builder",
+          "exemplar": false,
+          "expr": "redhat_appstudio_buildservice_simple_build_pipeline_creation_time_count",
```

Comment:
So for this example, metric `simple_build_pipeline_creation_time` is exported from build-service controller, and does `redhat_appstudio_buildservice_simple_build_pipeline_creation_time_count` map to that metric?

Does its format follow some Grafana or Promethues expression spec?

Is part `redhat_appstudio_buildservice_` fixed and defined for build-service specifically?

Does part `_count` show the calculation on the collected data of the metric?

Reply:
"Does its format follow some Grafana or Promethues expression spec?"
not sure what you mean, I don't know much about grafana of prometheus

but in the cluster our metric : simple_build_pipeline_creation_time
has to be queried as : redhat_appstudio_buildservice_simple_build_pipeline_creation_time

_count calculates number of metrics
_sum does sum for them
those are prometheus/grafana thingies